assets/js/payplug-integrated-payments.js

```diff
@@ -205,15 +205,19 @@ var IntegratedPayment = {
 				IntegratedPayment.props.return_url = response.redirect;
 			},
 			complete: function(response) {
-				if (response.responseJSON.result != "failure") {
-					if (IntegratedPayment.oneClickSelected()) {
-						return;
+				debugger;
+				if (response.responseJSON != null) {
+					if (response.responseJSON.result != "failure") {
```

Comment:
response.responseJSON != null && response.responseJSON.result != "failure" instead

Reply:
added a seperate if so we can add in the future an else to log the specific error because response.responseJSON comes from woocommerce and response.responseJSON.result comes from payplug